### PR TITLE
Add numeric type check for 'id' column in PostgreSQL sequence fix

### PR DESCRIPTION
**Pull request by almontasser**
This fixes the seed function `fixPostgresSequence`, it causes an error when executed on id fields of type uuid on postgres

### CODE REVIEW

**Review by bobbyiliev — APPROVED** [review]
I think that this looks good! Tagging @tnylea for a second review.

**Comment by tnylea** [comment]
Awesome. Thanks for the PR. I'll get this merged in and tag a new version after a few other PR's are merged in as well 👍

**Pull request merged**
